packages/combo-box/test/typings/combo-box.types.ts

```diff
@@ -69,41 +81,98 @@ narrowedComboBox.addEventListener('selected-item-changed', (event) => {
   assertType<TestComboBoxItem | null | undefined>(event.detail.value);
 });
 
-/* ComboBoxLightElement */
-const genericComboBoxLightElement = document.createElement('vaadin-combo-box-light');
-assertType<ComboBoxLight>(genericComboBoxLightElement);
+// ComboBox properties
+assertType<() => boolean>(narrowedComboBox.checkValidity);
+assertType<() => boolean>(narrowedComboBox.validate);
+assertType<() => void>(narrowedComboBox.close);
+assertType<() => void>(narrowedComboBox.open);
+assertType<boolean | null | undefined>(narrowedComboBox.autoOpenDisabled);
+assertType<boolean | null | undefined>(narrowedComboBox.opened);
+assertType<boolean>(narrowedComboBox.invalid);
+assertType<HTMLElement | null | undefined>(narrowedComboBox.focusElement);
+assertType<boolean>(narrowedComboBox.disabled);
+assertType<boolean>(narrowedComboBox.clearButtonVisible);
+assertType<string | null | undefined>(narrowedComboBox.errorMessage);
+assertType<string>(narrowedComboBox.placeholder);
+assertType<string | null | undefined>(narrowedComboBox.helperText);
+assertType<boolean>(narrowedComboBox.readonly);
+assertType<string | null | undefined>(narrowedComboBox.label);
+assertType<string>(narrowedComboBox.value);
+assertType<boolean>(narrowedComboBox.required);
+assertType<string>(narrowedComboBox.name);
```

Comment:
**note:** These properties are also candidates to be tested:

- `pattern`
- `preventInvalidInput`
- `itemIdPath`
- `itemValuePath`
- `itemLabelPath`
- `selectedItem`
- `filter`
- `loading`
- `filteredItems`
- `allowCustomValue`
- `items` 
- `autoselect`
- `title`
- `autofocus`

**idea:** We might also consider creating a sort of assert helpers for mixins:

```js
assertValidateMixin(narrowedCombobox);
```

Reply:
Added all of them except `title` which is a native attribute (we only have it for backwards compatibility).